src/admin/components/website/additionalComponents/LabeledInput.tsx

```diff
@@ -0,0 +1,71 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+import * as React from 'react';
+import { InputError } from 'admin/components/form/InputError';
+
+export interface ILabeledInput {
+	onSubmit: () => void;
+	errorText: string | boolean;
+	onEdit: (e) => void;
+	value: string;
+	touched: boolean;
+	label: any;
```

Comment:
Why label has 'any' type?

Reply:
It can be either text or a JSX component, but you are right, fixing it.